evap/evaluation/tests/test_commands.py

```diff
@@ -364,6 +364,31 @@ def test_dont_remind_already_voted(self):
         self.assertEqual(mock.call_count, 0)
         self.assertEqual(len(mail.outbox), 0)
 
+    def test_remind_only_once_evaluation_started_yesterday(self):
```

Comment:
I think this test can be slightly nicer if structured like this:
```python
user = ...
recent_evaluation = baker.make(Evaluation, ...)
management.call_command(...)
assertNotCalled()

old_evaluation = baker.make(Evalaution, ...)
management.call_command(...)
assertCalled(..., due_evaluations=[recent_evaluation, old_evaluation])
```

With the current test, it is a bit hard to see that we are actually also testing that user1 does not receive an email, whereas with the above it is clear. Additionally, we also want to check that when an email is sent, it also includes recent evaluations